src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.js

```diff
@@ -88,34 +81,38 @@ const convertInclude = async (includeBlock, ...restParams) => {
     return null;
   }
 
-  // This determines if the href has a leading '/'
-  const hrefTypePostion = () => (href.indexOf('/') === 0 ? 1 : 0);
-
-  // This checks if the supportedType is in the correct position of the href
-  const hrefIsSupported = () => supportedType =>
-    href.startsWith(supportedType, hrefTypePostion());
-
-  // This extracts the type from the href
-  const typeExtraction = Object.keys(supportedTypes).find(
-    hrefIsSupported(href),
-  );
+  const { includeType, classification } = includeClassifier({ href, pathname });
 
-  // This determines if the type is supported and returns the include type name
-  const includeType = supportedTypes[typeExtraction];
-  if (!includeType) {
+  if (
```

Comment:
this reads fine but at the time I saw this initially, there was 2 if blocks, not sure if this reads better
```
  const loggerClassifier = {
    'not-supported': INCLUDE_UNSUPPORTED,
    'vj-include-not-supporting-amp': INCLUDE_UNSUPPORTED,
  };

  if (loggerClassifier[classification]) {
    logger.info(loggerClassifier[classification], {
      type,
      classification,
      url: href,
    });
    return null;
  }
```

Reply:
Another variant 
```javascript
const unsupportedClassifications = [ 'not-supported', 'vj-include-not-supporting-amp' ];
if(unsupportedClassifications.includes(classification)) {
...
```